storage/sqlite3/nodereputation_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package nodereputation
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	proto "storj.io/storj/protos/nodereputation"
+)
+
+func TestCreateTable(t *testing.T) {
+	testDatabaseName := "./TestCreateTable.db"
+	StartDB(testDatabaseName)
+	os.Remove(testDatabaseName)
+}
+
+func TestCreateNode(t *testing.T) {
+	testDatabaseName := "./TestCreateNode.db"
+	db, _ := StartDB(testDatabaseName)
+
+	CreateNewNodeRecord(db, "Alice")
+
+	os.Remove(testDatabaseName)
+}
+
+func TestSelectReputation(t *testing.T) {
+	testDatabaseName := "./TestSelectReputation.db"
+	db, _ := StartDB(testDatabaseName)
+
+	CreateNewNodeRecord(db, "Alice")
+
+	_, err := GetRep(db, "Alice")
+	if err != nil {
+		panic(err)
```

Comment:
may be also a helpful log message as well? before you panic, may be put a log message?. I noticed you have a lot of panics. I won't comment on every one. What are your thoughts?